Sources/StreamChat/WebSocketClient/Events/UserEvents.swift

```diff
@@ -200,16 +200,34 @@ struct UserBannedEventDTO: EventWithPayload {
     }
 }
 
-public struct UserGloballyUnbannedEvent: UserSpecificEvent {
-    var userId: UserId
-    var createdAt: Date?
-    var payload: Any
+/// Triggered when user is removed from global ban.
+public struct UserGloballyUnbannedEvent: Event {
```

Comment:
Not used in SDK, currently. When `user.unbanned` comes without `cid` - the user was unbanned  `globally`.